RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ImportMnemonicControllingAccounts.swift

```diff
@@ -18,13 +18,33 @@ public struct ImportMnemonicControllingAccounts: Sendable, FeatureReducer {
 		public var isMainBDFS: Bool
 
 		public init(
-			entitiesControlledByFactorSource: EntitiesControlledByFactorSource,
+			entitiesControlledByFactorSource ents: EntitiesControlledByFactorSource,
 			isMainBDFS: Bool
 		) {
 			self.isMainBDFS = isMainBDFS
-			self.entitiesControlledByFactorSource = entitiesControlledByFactorSource
+			self.entitiesControlledByFactorSource = ents
+
+			let accounts: IdentifiedArrayOf<Profile.Network.Account> = switch (ents.babylonAccounts, ents.olympiaAccounts) {
+			case let (.some(babylonAccounts), _):
+				// We prefer Babylon, always.
+				babylonAccounts.rawValue
+			case let (nil, .some(olympiaAccounts)):
+				olympiaAccounts.rawValue
+			case (nil, nil):
```

Comment:
I'd say that if you use `.some`(which I prefer not to use), then the counterpart is `.none` rather than `nil`.

Reply:
ha I used `.none` but changed to `nil` since I thought you always *unconditionally* preferred `nil` :D

Reply:
So this is what I prefer:

```swift
switch anOptionalEnum {
case .enumCase: ...
case  nil: ...
}
```
and
```swift
switch anOptionalValue {
case let value?: ...
case  nil: ...
}
```